src/e2e/puppeteer/helpers/waitForEditable.ts

```diff
@@ -7,7 +7,7 @@ interface Options {
 /**
  * Wait for editable div that contains the given value.
  */
-const waitForEditable = async (page: Page, value: string, { timeout }: Options = { timeout: 6000 }) =>
+const waitForEditable = async (page: Page, value: string, { timeout }: Options = { timeout: 60000 }) =>
```

Comment:
The increased timeout was added temporarily for debugging purposes to investigate a specific issue. I don't anticipate needing more than 6 seconds in normal scenarios.

Reply:
Got it! Sounds good.

Just make sure to add a comment on the PR for changes that you are still testing, or switch the PR to a draft. Otherwise we will assume that it is ready for review.